integration-tests/src/endpoints/feeds.py

```diff
@@ -129,21 +128,21 @@ def test_feeds_with_status(self):
                     feed["status"] == status
                 ), f"Expected status '{status}', got '{feed['status']}'."
 
-    def test_filter_by_country_code(self):
-        """Test feed retrieval filtered by country code"""
-        df = pandas.concat([self.gtfs_feeds, self.gtfs_rt_feeds], ignore_index=True)
-        country_codes = self._sample_country_codes(df, 20)
-        task_id = self.progress.add_task(
-            "[yellow]Validating feeds by country code...[/yellow]",
-            total=len(country_codes),
-        )
-        for i, country_code in enumerate(country_codes):
-            self._test_filter_by_country_code(
-                country_code,
-                "v1/feeds",
-                task_id=task_id,
-                index=f"{i + 1}/{len(country_codes)}",
-            )
+    # def test_filter_by_country_code(self):
```

Comment:
picky: this can be disabled with 
```
    @pytest.mark.skip(reason="This test is expected to fail until API location issue is implemented")
    def test_filter_by_country_code(self):
```

Reply:
As we do not use pytest to run the integration tests, ill leave this commented for now but i'll address the changes as part of #622

Reply:
@cka-y Is it time to put back these tests?

Reply:
#662 needs to be addressed first